web/src/config/client_swagger.json

```diff
@@ -1014,37 +973,16 @@
 			"summary": "bucket  delete: Accessible to S3 account only",
 			"responses": {
 				"200": {
-					"description": "OK"
-				},
-				"201": {
-					"description": "No Content"
+					"description": "OK: Bucket name deleted successfully"
 				},
 				"400": {
-					"description": "Bad Request"
+					"description": "Bad Request: Request body is missing or Username or password is missing"
```

Comment:
incorrect message ,"username password" not in context

Reply:
Resolved